backend/canvas_app_explorer/views.py

```diff
@@ -1,15 +1,33 @@
-from django.shortcuts import render
-
+from canvas_oauth.oauth import get_oauth_token
+from django.conf import settings
 from rest_framework import permissions, viewsets
+from rest_framework.request import Request
 
 from backend.canvas_app_explorer import models, serializers
-
+from .canvas_lti_manager import CanvasLtiManager
 
 class LTIToolViewSet(viewsets.ModelViewSet):
     """
-    API endpoint that allows users to be viewed or edited.
+    API endpoint that lists LTI tools available in the course context
     """
-    queryset = models.LtiTool.objects.all()
     serializer_class = serializers.LtiToolSerializer
+    permission_classes = [permissions.DjangoModelPermissions]
+    queryset = models.LtiTool.objects.all()
+
+    def get_serializer_context(self):
```

Comment:
I don't think I've ever used this before, is this something with DRF? Like to pass data back?

Reply:
I had never either. It's mentioned lightly in the documentation here: https://www.django-rest-framework.org/api-guide/generic-views/

I think this SO post gave me the idea to use it: https://stackoverflow.com/a/53863873 Again, I'm still not positive this is the best way to do it.